stripe/src/test/java/com/stripe/android/StripeTest.java

```diff
@@ -248,6 +253,204 @@ public void createBankAccountTokenSynchronous_withValidBankAccount_returnsToken(
     }
 
     @Test
+    public void createSourceSynchronous_withBitcoinParams_passesIntegrationTest() {
+        Stripe stripe = new Stripe(mContext);
+        SourceParams bitcoinParams = SourceParams.createBitcoinParams(1000L, "usd", "abc@def.com");
+        try {
+            Source bitcoinSource =
+                    stripe.createSourceSynchronous(bitcoinParams, FUNCTIONAL_SOURCE_PUBLISHABLE_KEY);
+            assertNotNull(bitcoinSource);
+            assertNotNull(bitcoinSource.getId());
+            assertNotNull(bitcoinSource.getClientSecret());
+            assertEquals(Source.BITCOIN, bitcoinSource.getType());
+            assertEquals(1000L, bitcoinSource.getAmount().longValue());
```

Comment:
Adding. Note that I'm (presently) purposefully avoiding testing the contents of the source-type-specific responses, because I'm worried that they're not super-stable (given the lack of documentation). (Currency is not one of those).